src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.event.Event;
+
+
+/**
+ * An UI component that displays information of an {@code Event}.
+ */
+public class EventCard extends UiPart<Region> {
+
+    private static final String FXML = "EventListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Event event;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label eventTitle;
+    @FXML
+    private Label date;
+    @FXML
+    private Label time;
+    @FXML
+    private Label purpose;
+    @FXML
+    private Label id;
+
+
+    /**
+     * Creates a {@code EventCode} with the given {@code Event} and index to display.
```

Comment:
Is this supposed to be {@code EventCard}? I've noticed that PersonCard uses PersonCode too, is this a type on their end as well?
```suggestion
     * Creates a {@code EventCard} with the given {@code Event} and index to display.
```

Reply:
Yup this looks like a typo because the code `{@code EventCode}` is similar to GitHub MarkDown's Add Code feature which converts a text into a code text.